bpf/Makefile.bpf

```diff
@@ -3,11 +3,14 @@
 
 FLAGS := -I$(ROOT_DIR)/bpf/include -I$(ROOT_DIR)/bpf -D__NR_CPUS__=$(shell nproc) -O2 -g
 
-CLANG_FLAGS := ${FLAGS} -target bpf -emit-llvm
+CLANG_FLAGS := ${FLAGS} -target bpf -std=gnu89 -nostdinc -emit-llvm
```

Comment:
Do we need the same flags in `test/bpf/Makefile`?

Reply:
Yeah probably makes sense, will add, the CFLAGS lack also few other things in there.